resources/knative-serving-crds-init/README.md

```diff
@@ -0,0 +1,8 @@
+# Knative Serving CRDs
+
+## Overview
+
+This module is still experimental.
+This chart includes [knative-serving-crds](https://github.com/knative/docs/tree/master/docs/serving) release files.
+
+We need to install crds before installing the actual resources since its a dependency for the resources being installed
```

Comment:
```suggestion
Install CRDs before installing the actual resources as they depend on one another. 
```
But also question: do the actual resources depend on the CRDs (i understand it this way). If so in what way?

Reply:
This char is deleting the old crds if present. This is for upgrade case when we are upgrading the crds.

Reply:
fixed.